tests/s2_inference/embeddings_reference/info.txt

```diff
@@ -0,0 +1,7 @@
+16/10/24 - All embeddings were generated with:
```

Comment:
It's good that we have this information. But should it be part of the embedding file? This directory could also contain other embeddings in the future. If you have the capacity, you can even create a pydantic class for the embedings.